### ISSUE TITLE
mashtree_bootstrap.pl reading fasta files as commands

### ISSUE THREAD

**Issue by noaheb98** [lskatz/mashtree#89]
Hi, I'm trying to run mashtree_bootstrap.pl on a set of fasta files:

`mashtree_bootstrap.pl --reps 1000 --numcpus 12 /home/files/*.fasta -- --min-depth 0  > out.tre`

But I'm getting a list of errors in which it displays the last bit of each fasta file name and stating that the command cannot be found:

sh: line 1: -.fasta: command not found

Running mashtree on the directory of fasta files worked fine, but not for bootstrapping. Should the input files be different than what's used for mashtree?

**Comment by lskatz** [comment]
Do you... Have semicolons, dashes, and/or semicolons in your filenames? Can you run `ls -lh /home/files`?

**Comment by noaheb98** [comment]
Yes some have dashes at the end before .fasta and some end in +.fasta.

**Comment by noaheb98** [comment]
> Yes some have dashes at the end before .fasta and some end in +.fasta.

Each error that comes up shows the end of each fasta file, such as :
sh: line 1: -.fasta: command not found
sh: line 1: +.fasta: command not found

**Comment by noaheb98** [comment]
> Do you... Have semicolons, dashes, and/or semicolons in your filenames? Can you run `ls -lh /home/files`?

Here's an example for one of them when I run ls -lh:

-rw-r--r-- 1 nbrown43 nbrown43  18K Sep 13 14:16 's87_s00830|-.fasta'

I have about 1500 fasta files.

**Comment by lskatz** [comment]
Please remove the `|` characters from your filenames
